v2.2/recommended-production-settings.md

```diff
@@ -39,54 +39,89 @@ Term | Definition
-        {{site.data.alerts.end}}
+For best performance:
+
+- Use NVMe SSDs over HDDs.
+- To simulate a bare metal deployment where machines are directly connected to one physical disk each, create and mount a local SSD using a SCSI interface.
+- Use more powerful nodes. Adding more CPU is usually more beneficial than adding more RAM.
+- To calculate IOPS, use [sysbench](https://github.com/akopytov/sysbench). If IOPS decreases, add more nodes to your cluster to add IOPS.
+
+#### Resilience recommendations
+
+For more resilient clusters:
+
+- Use many smaller nodes instead of fewer larger ones. Recovery from a failed node is faster when data is spread across more nodes. We recommend using 8 cores (i.e., 16 threads), one per node.
+- Use [zone configs](configure-replication-zones.html) to increase the replication factor from 3 (the default) to 5 (across at least 5 nodes). This is especially recommended if you are using local disks with no RAID protection rather than a cloud providers' network-attached disks that are often replicated underneath the covers, because local disks have a greater risk of failure. You can do this for the [entire cluster](configure-replication-zones.html#edit-the-default-replication-zone) or for specific [databases](configure-replication-zones.html#create-a-replication-zone-for-a-database), [tables](configure-replication-zones.html#create-a-replication-zone-for-a-table), or [rows](configure-replication-zones.html#create-a-replication-zone-for-a-table-or-secondary-index-partition) (enterprise-only).
+    {{site.data.alerts.callout_danger}}
+    {% include {{page.version.version}}/known-limitations/system-range-replication.md %}
+    {{site.data.alerts.end}}
 
 ### Cloud-specific recommendations
 
 Cockroach Labs recommends the following cloud-specific configurations based on our own internal testing. Before using configurations not recommended here, be sure to test them exhaustively.
 
+In general, we have found CockroachDB to perform better on AWS over GCE.
+
 #### AWS
 
-- Use `m` (general purpose), `c` (compute-optimized), or `i` (storage-optimized) [instances](https://aws.amazon.com/ec2/instance-types/). For example, Cockroach Labs has used `m3.large` instances (2 vCPUs and 7.5 GiB of RAM per instance) for internal testing.
-- **Do not** use ["burstable" `t2` instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/t2-instances.html), which limit the load on a single core.
-- Use [Provisioned IOPS SSD-backed (io1) EBS volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html#EBSVolumeTypes_piops) or [SSD Instance Store volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html).
+- Use `m` (general purpose), `c` (compute-optimized), or `i` (storage-optimized) [instances](https://aws.amazon.com/ec2/instance-types/). For example, Cockroach Labs has used `m5d.xlarge` instances (4 vCPUs and 16 GiB of RAM per instance) for internal testing.
```

Comment:
Also need to specify that the D in `m5d.xlarge` is SSD. Also, I'd use the C series and I'd use `c5d.4xlarge` to show the 16 cpu